crates/bin/pd/src/migrate/testnet76.rs

```diff
@@ -95,16 +112,20 @@ pub async fn migrate(
         tracing::info!(?post_upgrade_root_hash, "post-upgrade root hash");
 
         (
-            start_time.elapsed().expect("start time is set"),
+            start_time.elapsed().expect("start time not set"),
```

Comment:
@cronokirby fwiw, the convention is that you write why you expect the `Result/Option` to be `Ok/Some`